documentation/optee_design.md

```diff
@@ -611,68 +431,68 @@ struct optee_header {
         uint32_t paged_size;
 };
 ```
+
 The header is only used by the loader of OP-TEE, not OP-TEE itself. To
-initialize OP-TEE the loader loads the complete binary into memory and
-copies what follows the header and the following ```init_size``` bytes to
-```(init_load_addr_hi << 32 | init_load_addr_lo)```. ```init_mem_usage``` is
-used by the loader to be able to check that there's enough physical memory
-available for OP-TEE to be able to initialize at all. The loader supplies
-the address of the first byte following what wasn't copied in r0/x0 and jumps
-to the load address to start OP-TEE.
+initialize OP-TEE the loader loads the complete binary into memory and copies
+what follows the header and the following `init_size` bytes to
+`(init_load_addr_hi << 32 | init_load_addr_lo)`. `init_mem_usage` is used by the
+loader to be able to check that there's enough physical memory available for
+OP-TEE to be able to initialize at all. The loader supplies the address of the
+first byte following what wasn't copied in r0/x0 and jumps to the load address
+to start OP-TEE.
 
 ## Initializing the pager
-The pager is initialized as early as possible during boot in order to
-minimize the "init" area. As a consequence of this the old initialization
-has been broken up a bit since the pager initialization depends on some
-internal APIs being available.
-
-The global variable `tee_mm_vcore` describes the virtual memory range that
-is covered by the level 2 translation table supplied to `tee_pager_init()`.
+The pager is initialized as early as possible during boot in order to minimize
+the "init" area. As a consequence of this the old initialization has been broken
+up a bit since the pager initialization depends on some internal APIs being
+available. The global variable `tee_mm_vcore` describes the virtual memory range
```

Comment:
Remove "As a consequence of this the old initialization has been broken up a bit since the pager initialization depends on some internal APIs being available."